frontend-student/src/hooks/useApi.jsx

```
@@ -1,12 +1,12 @@
-import useAuth from "../context/authContext";
+import useAuth from '../context/authContext';
 
 export default function useApi() {
   const { user } = useAuth();
   const fetchData = async (apiFunction, params) => {
-    console.log(apiFunction, params);
+    // console.log(apiFunction, params);
     let newParams = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'newParams' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)